src/main/java/net/masterthought/cucumber/generators/AbstractPage.java

```diff
@@ -79,6 +80,7 @@ private Properties buildProperties() {
         props.setProperty("resource.loader", "class");
         props.setProperty("class.resource.loader.class", ClasspathResourceLoader.class.getCanonicalName());
         props.setProperty("runtime.log", new File(configuration.getReportDirectory(), "velocity.log").getPath());
+        props.setProperty("eventhandler.referenceinsertion.class", "org.apache.velocity.app.event.implement.EscapeHtmlReference");
```

Comment:
Having looked at [flexmark](https://github.com/vsch/flexmark-java)  I think it would be fairly simple to add. Though I also think that should be done on another pull request.